### PR TITLE
Integration test

### PR DESCRIPTION
**Pull request by jdkandersson**
Adds 2 integration tests:

- That the workload becomes active
- That it is possible to communicate with another charm through the ingress relation

### CODE REVIEW

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    # Check that both ingress and hello-kubecon are active
+    assert ops_test.model.applications[app_name].units[0].workload_status == ActiveStatus.name
+    assert (
+        ops_test.model.applications[hello_kubecon_app_name].units[0].workload_status
+        == ActiveStatus.name
+    )
+
+    # Add required relations
+    await ops_test.model.add_relation(hello_kubecon_app_name, app_name)
+    await ops_test.model.wait_for_idle(status="active")
```
Comment (arturo-seijas) [inline tests/integration/conftest.py:51]: Maybe it'd be better to use ActiveStatus.name instead of "active"

**Review by arturo-seijas — CHANGES_REQUESTED** [review]

**Review by weiiwang01 — APPROVED** [review]

File: tests/integration/conftest.py (hunk 1 above)
Comment (jdkandersson) [reply tests/integration/conftest.py]: Thanks for pointing this out, fixed

**Review by weiiwang01 — APPROVED** [review]
LGTM, thanks!

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Pull request merged**